File: ex02.c
#include <stdio.h>
#include <math.h>

double calcular_pi(int n) {
    double soma = 0;
    for (int i = 1; i <= n; i++) {
        double termo = pow(-1, i-1) / pow(2*i - 1, 3);
        soma += termo;
    }
    return pow(soma * 32, 1.0/3);
}

int main() {
    int n;
    printf("digite o número de termos da série:");
    scanf("%d", &n);
    double pi = calcular_pi(n);
    printf("o valor aproximado de pi com %d termos da série é: %.10f\n", n, pi);
    return 0;
}

File: ex02.java
import java.util.Scanner;

public class ex02 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("digite o número de termos:");
        int n = scanner.nextInt();

        double s = 0.0;
        for (int i = 1; i <= n; i++) {
            s += 1.0 / (Math.pow(i, 3));
        }

        double pi = Math.pow((s * 32), 1.0 / 3);

        System.out.printf("o valor de pi com %d termos é: %.5f\n", n, pi);
    }
}
